frontend/app/src/components/profitloss/ReportsTable.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <v-card>
+    <v-card-title>
+      <card-title> {{ $t('profit_loss_reports.title') }}</card-title>
+    </v-card-title>
+    <v-card-text>
+      <v-sheet outlined rounded>
+        <data-table
+          show-expand
+          single-expand
+          sort-by="created"
+          item-key="identifier"
+          :items="index.reports"
+          :headers="headers"
+        >
+          <template #item.name="{ item }">
+            {{ item.name }}
+          </template>
+          <template #item.created="{ item }">
+            <date-display :timestamp="item.created" />
+          </template>
+          <template #item.start="{ item }">
+            <date-display :timestamp="item.startTs" />
+          </template>
+          <template #item.end="{ item }">
+            <date-display :timestamp="item.endTs" />
+          </template>
+          <template #item.sizeOnDisk="{ item }">
+            {{ item.sizeOnDisk }}
+          </template>
+        </data-table>
+      </v-sheet>
+    </v-card-text>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { ReportCache } from '@rotki/common/lib/reports';
+import { Component, Mixins } from 'vue-property-decorator';
+import { DataTableHeader } from 'vuetify';
+import { mapActions, mapGetters, mapState } from 'vuex';
+import DataTable from '@/components/helper/DataTable.vue';
+import CardTitle from '@/components/typography/CardTitle.vue';
+import StatusMixin from '@/mixins/status-mixin';
+import { ReportActions } from '@/store/reports/const';
+
+@Component({
```

Comment:
Any chance we could make this closer to how the https://github.com/rotki/rotki/blob/3a9a88363ef0724388f7c15956de8b34728fda89/frontend/app/src/views/history/Transactions.vue#L83 component looks like? for new components I am already using the composition-api instead of the class components approach, in order to ease the move to vue 3 later

Reply:
for the computed properties from the state I should have some helpers in a composables folder with names similar to the vuex module